models/maisi_ct_generative/scripts/sample.py

```diff
@@ -327,13 +496,20 @@ def __init__(
         controllable_anatomy_size,
         image_output_ext=".nii.gz",
         label_output_ext=".nii.gz",
-        quality_check_args=None,
+        real_img_median_statistics="./configs/image_median_statistics.json",
```

Comment:
Hi @Can-Zhao , here I  think we should use an arg,  and cannot hard code to this path. CI failed and error shows that cannot find this file.
We may need sth like:
"real_img_median_statistics": "$@bundle_root + '/configs/image_median_statistics.json'"

Reply:
I pushed a commit to resolve it

Reply:
Thank you so much!